### DISCUSSION TITLE
How to change some "hidden" key binding in QTerminal? For example, to change "scroll one line up" from shift-up to another

### DISCUSSION THREAD

**Discussion by traditionalgithub** [lxqt/lxqt#2041] [Q&A]
##### QTerminal related information

* Distribution & Version:  Ubuntu 20.04 LTS
* Kernel:  Linux  5.4.0-80-generic
* Qt Version: 5.12.8
* qtermwidget Version:  
* lxqt-build-tools Version:  x86_64 prebuilt from Ubuntu  repo
* Package version: QTerminal 0.14.1

I am currently using QTerminal the version described as above (prebuilt from the Ubuntu repository) and  I wish to change the keybinding of "scrolling one line up" from shift-up to another one or disable this binding (this binding is a shortcut to change a task priority in org-mode), but i can't find this one in the preference setting, neither in ~/.config/qterminal.org/qterminal.ini. Is this binding described in somewhere else?

**Comment by yan12125** [comment]
Currently you need to modify /usr/share/qtermwidget5/kb-layouts/default.keytab (the path may be different for different distros) and remove the following line
```
key Up      +Shift-AppScreen : scrollLineUp
```
It is planned to allow overriding system keytab files: https://github.com/lxqt/qterminal/issues/680
